TwilioVerifySDKTests/TwilioSecurity/Sources/Storage/SecureStorageTests.swift

```diff
@@ -163,6 +163,53 @@ class SecureStorageTests: XCTestCase {
       )
     }
   }
+
+  func testGet_valueExistsForMultipleTries_shouldReturnValueIfAnyOfTheTriesSucceed() throws {
+    // Given
+    let expectation = expectation(description: "Test multiple tries of storage.Get")
+    guard let expectedData = "data".data(using: .utf8) else {
+      XCTFail("failed to encrypt data")
+      return
+    }
+
+    let key = "key"
+    var data: Data?
+    keychain.error = TestError.operationFailed
+    keychain.keys = [expectedData as AnyObject, expectedData as AnyObject , expectedData as AnyObject]
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/b8f1e4c4694ac4732d318a7f2b21e7a8ef80870b/TwilioVerifySDKTests/TwilioSecurity/Sources/Storage/SecureStorageTests.swift#L178"></span>There should be no space before and one after any comma.<br />
<code>comma</code> <code>SecureStorageTests.swift:178</code></td>
    </tr>
  </tbody>
</table>